Library/Homebrew/exceptions.rb

```diff
@@ -360,8 +360,12 @@ def initialize(formula, cmd, args, env)
     @cmd = cmd
     @args = args
     @env = env
-    pretty_args = args.map { |arg| arg.to_s.gsub " ", "\\ " }.join(" ")
```

Comment:
`pretty_args = Array(args).map ...` might be simpler as it can avoid the `if`.

Reply:
But then it prints an extra space in that case… not sure how much we care about that

Reply:
Haven't tested it yet, but given `args` and `env` default values of `[]` and `{}` in the exception's constructor would also probably work here.